### PR TITLE
[SECURITY]: fix the problem with no access to endpoints

### PR DESCRIPTION
**Pull request by IhorKalaur**

### CODE REVIEW

**Review by Denis-Balako — APPROVED** [review]
LGTM!

File: src/main/resources/db/changelog/changes/09-modify-roles-into-roles-table.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+                  name: name
+                  value: ROLE_MANAGER
+            where: name = 'MANAGER'
+        - update:
+            tableName: roles
+            columns:
+              - column:
+                  name: name
+                  value: ROLE_CUSTOMER
+            where: name = 'CUSTOMER'
```
Comment (Someboty) [inline src/main/resources/db/changelog/changes/09-modify-roles-into-roles-table.yaml:19]: You lost empty line here

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -3,6 +3,14 @@ bot.token=${BOT_TOKEN}
...
+spring.datasource.password=${DB_PASSWORD}
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.show-sql=true
+spring.jpa.open-in-view=false
+spring.liquibase.enabled=true
 
-jwt.expiration=30000
+jwt.expiration=300000000
 jwt.secret=MayTheForceBeWithYou.ThereIsNoEmotion,ThereIsPeace
```
Comment (Someboty) [inline src/main/resources/application.properties:16]: I think we should hide (and change) it too

**Review by Someboty — APPROVED** [review]
I don't want to block you, but it will be great, if you add this:

**Pull request merged**